### DISCUSSION TITLE
where to find out advanced chatgpt prompts?

### DISCUSSION THREAD

**Discussion by uniluxs** [community/community#68279] [Other Feature Feedback, Questions, & Ideas]
if you know where we can find advanced Chatgpt prompts,  share it

**Comment by gy-hou** [comment]
Platform: Youtube/Discord/Reddit/Twitter/ 
Website: https://flowgpt.com/.   GitHub Projects

**Comment by tuves** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
